appengine/pom.xml

```diff
@@ -75,8 +75,10 @@
     <dependency>
       <groupId>com.google.auth</groupId>
       <artifactId>google-auth-library-oauth2-http</artifactId>
-      <type>test-jar</type>
+      <version>${project.version}</version>
```

Comment:
Do we need the version? I think it is already configured in the parent pom?

Reply:
I removed it, thanks for the catch.